docs/workflows/copy.rst

```diff
@@ -0,0 +1,33 @@
+Copy RPM content between repositories
+=====================================
+
+If you want to copy RPM content from one repository into another repository, you can do so.
+
+
+.. _copy-workflow:
+
+Copy workflow
+-------------
+
+You can use the copy endpoint to copy RPM-related content present in one repository / repository version to another repository. If you specify a repository, then the latest repository version for that repository will be used.
```

Comment:
nitpick. Can all the text lines be line wrapped at 100 chars? It's ok to me if some of the links or quoted items go over 100 lines.